### PR TITLE
fix: person credential workflow fix

### PR DESCRIPTION
**Pull request by amanji**
Wallet will now route to person credential offer when received by IDIM.

### CODE REVIEW

File: app/src/components/AddCredentialSlider.tsx [hunk 1]
```diff
@@ -21,24 +21,18 @@ const AddCredentialSlider: React.FC = () => {
...
+
   const credentials = [
     ...useCredentialByState(CredentialState.CredentialReceived),
     ...useCredentialByState(CredentialState.Done),
   ]
+
   const navigation = useNavigation()
   const [canUseLSBCredential] = useState<boolean>(true)
 
-  useEffect(() => {
```
Comment (wadeking98) [inline app/src/components/AddCredentialSlider.tsx:31]: any reason why we're removing this here?

Comment (amanji) [reply app/src/components/AddCredentialSlider.tsx:31]: Just organizing them all in one place

Comment (wadeking98) [reply app/src/components/AddCredentialSlider.tsx:31]: ah okay, I see it now

**Review by wadeking98 — APPROVED** [review]
Couple code smells to address but otherwise looks good

**Pull request merged**